receiver/azureeventhubreceiver/README.md

```diff
@@ -98,8 +98,10 @@ and the OpenTelemetry attributes.
 | time (required)                  | time_unix_nano (field)                 | 
 | identity (optional)              | azure.identity (attribute, nested)     |
 
-Note: JSON does not distinguish between fixed and floating point numbers. All
+Notes:
+* JSON does not distinguish between fixed and floating point numbers. All
 JSON numbers are encoded as doubles.
+* Additionally the field timestamp can be used instead of time.
```

Comment:
```suggestion
* Additionally the field `timestamp` can be used instead of `time`.
```
Formatting nit.

Do you want to add the `timestamp`/`timeStamp` field to the table above since it's supported now?

Also, would it be helpful to state in the README here that if both are included, `time` takes precedence over `timestamp`?

Reply:
I added the note as suggested. I was trying to make the least amount of change to the table above to avoid making confusing. Hopefully what I did makes sense.